networkmanager.te

```diff
@@ -443,10 +443,14 @@ optional_policy(`
     ssh_basic_client_template(NetworkManager, NetworkManager_t, system_r)
     term_use_generic_ptys(NetworkManager_ssh_t)
     modutils_domtrans_kmod(NetworkManager_ssh_t)
+    corenet_rw_tun_tap_dev(NetworkManager_ssh_t)
     dbus_connect_system_bus(NetworkManager_ssh_t)
     dbus_system_bus_client(NetworkManager_ssh_t)
-
+    init_rw_stream_sockets(NetworkManager_ssh_t)
```

Comment:
In the avc list, I see only getattr and ioctl reported. Did you reproduce it so that also other permissions were requested?

Reply:
this is a common access:

NetworkManager_ssh_t init_t:fd use;
NetworkManager_ssh_t init_t:unix_stream_socket { getattr ioctl read write };

all processes forked by systemd (directly or indirectly) need this. In this case networkmanager_ssh_t is probably executed by networkmanager_t which is executed by init_t, so even though networkmanager_ssh_t is not a direct decendant of init_t it still is indirect. I believe this event is triggered by the way systemd/journald intercept stdout (logging "feature")